include/ringmesh/io/geomodel_builder_gocad.h

```diff
@@ -89,7 +90,7 @@ namespace RINGMesh
         virtual void read_line() = 0;
 
     protected:
-        GEO::LineInput file_line_{ filename_ };
+        GEO::LineInput file_line_{ filename() };
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make this member variable "private". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS3656)